### PR TITLE
changes in ML track

### PR DESCRIPTION
**Pull request by kabirnagpal**
# #40

### CODE REVIEW

File: soa/tracks/ml/8.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+We'll learn About  
+
+- Gradient Boosting Algorithm  
+- Extreme Boosting Algorithm  
+- Ada Boost Algorithm  
+
+
+
+### Click [here](https://github.com/kabirnagpal/SoA-ML-14/blob/master/week%208.ipynb) to view the Jupyter-Notebook.
```
Comment (theSage21) [inline soa/tracks/ml/8.md:15]: Can we reserve headings for actual headings please?

File: soa/tracks/ml/8.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+### Click [here](https://github.com/kabirnagpal/SoA-ML-14/blob/master/week%208.ipynb) to view the Jupyter-Notebook.   
+If you don't have any Python Environment, you can also try the code in [Google Colab](https://colab.research.google.com/).  
+
+
+Question to be answered after you complete your notebook
+Which of the following algorithm are not an example of ensemble learning algorithm?
+
+A. Random Forest
```
Comment (theSage21) [inline soa/tracks/ml/8.md:22]: This is going to show up as a normal sentence. If you want numbered lists in markdown you'll need to use

```
1. something
2. something else
```

File: soa/tracks/ml/8.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- Extreme Boosting Algorithm  
+- Ada Boost Algorithm  
+
+
+
+### Click [here](https://github.com/kabirnagpal/SoA-ML-14/blob/master/week%208.ipynb) to view the Jupyter-Notebook.   
+If you don't have any Python Environment, you can also try the code in [Google Colab](https://colab.research.google.com/).  
+
+
+Question to be answered after you complete your notebook
```
Comment (theSage21) [inline soa/tracks/ml/8.md:19]: Please maintain sentence case. In this sentence for example, a full stop is missing.

File: soa/tracks/ml/1.md [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
-If you don't have any Python Environment, you can also try the code in [Google Colab](https://colab.research.google.com/).
+- [Numpy](https://numpy.org/)  
+- [Pandas](https://pandas.pydata.org/)  
+- [Matplotlib](https://matplotlib.org/)  
+
+Click [here](https://github.com/kabirnagpal/SoA-ML-14/blob/master/week%201.ipynb) to view the Jupyter-Notebook.  
+If you don't have any Python Environment, you can also try the code in [Google Colab](https://colab.research.google.com/).  
+
+
+How to get mean of each column in a Data Frame named `df`?
```
Comment (theSage21) [inline soa/tracks/ml/1.md:13]: Shouldn't we tell people that this question is to be answered after they have gone through the notebook? Like it is now, the question is a little confusing.

File: soa/tracks/ml/3.md [hunk 5]
```diff
@@ -13,7 +13,9 @@ This week we'll learn about:
 Click [here](https://github.com/kabirnagpal/SoA-ML-14/blob/master/week%203.ipynb) to view the Jupyter-Notebook.
 If you don't have any Python Environment, you can also try the code in [Google Colab](https://colab.research.google.com/)
 
-`mean_squared_error` is a method from which class in Sklearn?
+
+`mean_squared_error` is a method from which class in Sklearn?
```
Comment (theSage21) [inline soa/tracks/ml/3.md:17]: Is metrics a class in sklearn? I thought it was a module.

File: soa/tracks/ml/6.md [hunk 6]
```diff
@@ -6,18 +6,21 @@ Clustering is basically the task of dividing the population or data points into
 
 So in this week we are going to dive deep into the clustering and cover the following topics:
 
-- what is clustering
+- What is clustering
```
Comment (theSage21) [inline soa/tracks/ml/6.md:9]: Please follow normal english grammar. For example ending sentences with ? or .

**Comment by kabirnagpal** [comment]
Okay. I’ll look into the changes myself and give another pull request.

**Comment by theSage21** [comment]
You can just push changes to this branch and this pr will get updated.

When you're done you can request review again.

File: soa/tracks/ml/8.md (hunk 1 above)
Comment (kabirnagpal) [reply soa/tracks/ml/8.md:15]: I didn't get this issue. Rest has been resolved!

**Comment by theSage21** [comment]
The `###` is for headings. Let's use that for actual headings. 😊

File: soa/tracks/ml/8.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+# ML Track-Week 8
```
Comment (theSage21) [inline soa/tracks/ml/8.md:1]:
```suggestion
<a href='https://t.me/ml_code_for_100_days'><button>Discuss on telegram</button></a>
# ML Track-Week 8
```

**Pull request merged**